packages/KTextArea/KTextArea.vue

```diff
@@ -17,6 +17,7 @@
       <div class="text-on-input">
         <label
           :for="textAreaId"
+          v-bind="labelAttributes"
```

Comment:
Same comment here -- what does binding `labelAttributes` here do? Seemingly nothing?

Reply:
You could also do something like apply a `data-testid` to the label.